babel.config.js

```diff
@@ -51,8 +51,10 @@ module.exports = {
      * https://github.com/tc39/proposal-decorators
      **/
     ['@babel/plugin-proposal-decorators', { legacy: true }],
+    // The "loose" option must be the same for all three of these plugins.
     ['@babel/plugin-proposal-class-properties', { loose: true }],
     ['@babel/plugin-proposal-private-methods', { loose: true }],
+    ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
```

Comment:
Also need to add as a dep? Maybe in root/package.json? 🤷‍♂️

Reply:
I'll check; thought it was included in the preset but not sure 🤔

Reply:
Added it just to be sure for now